bin/install.exec

```diff
@@ -15,5 +15,5 @@ if [[ TARGET_INSTALL_REMOTE -eq 1 ]]; then
 
 	exec ${args[@]}
 else
-	exec sh -c "$@"
+	exec su -c "$@"
```

Comment:
Well, most users' usage of `install.exec` involves connecting to their device via SSH as `root` anyway, so... I think this change should be fine?